js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -1148,6 +1150,8 @@ webAuthnPolicyRpId=Relying party ID
 ldapRolesDnHelp=LDAP DN where roles of this tree are saved. For example, 'ou\=finance,dc\=example,dc\=org'.
 serviceAccount=Service account roles
 standardTokenExchangeEnabled=Standard Token Exchange
+jwtAuthorizationGrantEnabled=JWT Authorization Grant
+jwtAuthorizationGrantIdp=Select Allowed Identity Providers for JWT Authorization Grant
```

Comment:
```suggestion
jwtAuthorizationGrantIdp=Allowed Identity Providers for JWT Authorization Grant
```

This is the name of the option, so perhaps we do not need the word "select" beforehand in the name of the option? We do not use it for any other option as well.